src/inmanta/app.py

```diff
@@ -455,6 +455,13 @@ def export_parser_config(parser: argparse.ArgumentParser, parent_parsers: abc.Se
         "be used together with the --partial option.",
         action="append",
     )
+    parser.add_argument(
+        "--soft-delete",
```

Comment:
I'm still thinking if I can come up with a better name. This one was always meant as a placeholder, but perhaps it's not too bad.

Reply:
I couldn't come up with something better and it kinda grew on me.